src/firebaseWrapper.js

```diff
@@ -37,9 +37,31 @@ const logout = () => {
 /**
  * Messaging Functions
  */
-// const sendMessage = (conversationId, userId, message) => {};
+const sendMessage = (user, receiver, message) => {
```

Comment:
similar to the other comment here. change the interface to `(conversationId, userId, message) => {};`